client/src/js/services/SnisService.js

```diff
@@ -0,0 +1,18 @@
+angular.module('bhima.services')
+  .factory('SnisService', SnisService);
```

Comment:
Since this is in the module "bhima.services" it's better to use a service.  They are almost identical.  See #53.